IaC/AKS/README.md

```diff
@@ -417,11 +417,16 @@ envsubst < clusterissuer.yaml | kubectl apply -f -
 
 ## Deploy NGSA with Helm
 
-A helm chart is included for the reference application ([NGSA](https://github.com/retaildevcrews/ngsa)).
+The NGSA helm chart can either be manually installed as explained below or through a GitOps continuous deployment approach by following the instructions in ([NGSA-CD](https://github.com/retaildevcrews/ngsa-cd)).
 
 ```bash
+cd $HOME
+
+git clone git@github.com:retaildevcrews/ngsa-cd.git
+
+export CHART_REPO=$HOME/ngsa-cd
```

Comment:
have the user cd into the repo. and possibly save the location. they might not always clone to home.
maybe something like
```suggestion
cd ngsa-cd
export CHART_REPO=$pwd/ngsa-cd
```